index.js

```diff
@@ -0,0 +1,19 @@
+/*
+Copyright 2015 Colin Clark
+
+Licensed under the 3-Clause "New" BSD license.
+You may not use this file except in compliance with one these
+Licenses.
+
+You may obtain a copy of the 3-Clause "New" BSD License at
+https://github.com/colinbdclark/infusion-electron/raw/master/LICENSE.txt
+*/
+
+"use strict";
+
+var fluid = require("infusion");
+require("./src/app.js");
+require("./src/core.js");
+require("./src/window.js");
+
```

Comment:
I think this might be a good place for the two stray "requires" that you have within component material - 
perhaps something like electron.ipc = require("ipc") 
and electron.app = require("app")
I'm not clear on how these are resolved but it's increasingly appearing to me that this must be some kind of "electron magic" since I can't see how they could be resolvable by the normal node module rules.